test/test_abstract_operations.jl

```diff
@@ -115,6 +116,20 @@ for arch in archs
             end
         end
 
+        @testset "KernelFunctionOperations [$(typeof(arch))]" begin
+            trivial_kernel_function(i, j, k, grid) = 1
+            op = KernelFunctionOperation(trivial_kernel_function, grid)
+            @test op isa KernelFunctionOperation
+
+            less_trivial_kernel_function(i, j, k, grid, u, v) = @inbounds u[i, j, k] * v[i, j, k]
```

Comment:
Is it worth  having a test that specifies the architecture?  I noticed there were two functions and I gather these tests only use the one that doesn't specify the architecture.